docs/Limitations.md

```diff
@@ -0,0 +1,19 @@
+# Limitations of Azure DevOps Demo Generator
```

Comment:
I'll have some minor verbiage changes, but let's get this PR through and then I'll PR back to you with those changes...